server/routerlicious/packages/routerlicious-base/src/utils/sessionHelper.ts

```diff
@@ -89,22 +89,30 @@ async function updateExistingSession(
         ) {
             // Previous session was in a different location. Move to current location.
             // Reset logOffset, ordererUrl, and historianUrl when moving session location.
-            Lumberjack.info(
-                `Reset logOffset, ordererUrl, and historianUrl when switching cluster.`,
-                lumberjackProperties,
-            );
+            Lumberjack.info("Moving session", {
+                ...lumberjackProperties,
+                isSessionSticky,
+                documentLastAccessTime: document.lastAccessTime,
+                dateTime: Date.now(),
```

Comment:
Regarding the other commend where we talked about `dateTime`. You mentioned that `isSessionSticky` is calculated based on it. However, one important note is that the `Date.now()` being used here in line 96 will not be the same as `Date.now()` from line 78. So I would recommend doing the following, if you want to log the timestamp  used in the `isSessionSticky` computation:
1. Add a new line before line 77, like `const sessionStickyCalculationTimestamp = Date.now();`
2. In line 78, do: `? sessionStickyCalculationTimestamp - document.lastAccessTime < sessionStickinessDurationMs`
3. Here, in this Lumberjack, you would log the 3 things. E.g.
```ts
Lumberjack.info("Moving session", {
                ...lumberjackProperties,
                isSessionSticky,
                documentLastAccessTime: document.lastAccessTime,
                sessionStickyCalculationTimestamp,
                currentSessionStickinessDurationMs: sessionStickyCalculationTimestamp - document.lastAccessTime,
                sessionStickinessDurationMs
```

And maybe we can rename `sessionStickinessDurationMs` to `minimumSessionStickinessDurationMs`?

Reply:
Thanks. :) Actually, I thought the same idea. Made the fix.